phases/01-math-foundations/07-bayes-theorem/docs/en.md

```
@@ -410,22 +411,22 @@ Setup: you are testing two button colors. Variant A (blue) and variant B (green)
 3. **Posteriors.**
-   - A: Beta(1 + 50, 1 + 950) = Beta(51, 951). Mean = 0.051
-   - B: Beta(1 + 65, 1 + 935) = Beta(66, 936). Mean = 0.066
-4. **Decision.** Compute P(B > A) -- the probability that B's true conversion rate is higher than A's.
+   - A: $\text{Beta}(1 + 50, 1 + 950) = \text{Beta}(51, 951)$. Mean $= 0.051$
+   - B: $\text{Beta}(1 + 65, 1 + 935) = \text{Beta}(66, 936)$. Mean $= 0.066$
+4. **Decision.** Compute $P(B > A)$ -- the probability that B's true conversion rate is higher than A's.
 
-Computing P(B > A) analytically is hard. But Monte Carlo makes it trivial:
+Computing $P(B > A)$ analytically is hard. But Monte Carlo makes it trivial:
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)